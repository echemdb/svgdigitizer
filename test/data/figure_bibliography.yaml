# This file can exist with the "source" information only
# 'Figure description' is adapted based on SVG input
version: 1

source:
    doi: https://doi.org/10.1016/0039-6028(85)90985-9
    citationKey: cv_bibliography # AuthorName_YYYY_FirstWordTitle_pageNr. No prepositions. Fingers crossed that there is no duplicate.
    figure: 2b # depending on the subfigure labels use 1, 1b, or 1_3
    curve: First cycle # use a unique short description to identify the curve in the figure

curator:
    name: Albert Engstfeld
    orcid: https://orcid.org/0000-0002-9686-3948
    digitized: 2021-06-01 # YYYY-MM-DD

electrochemicalSystem:
    electrolyte:
        type: aq # aqueous = aq, non-aqueous=naq
        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
            - name: NaCl # replace name by something generic, i.e., for a complex organic compound
              sumFormula:  NaCl
              concentration:
                  value: 0.1
                  unit: M #[M, mM, µM, g kg-1, ...]
              type: salt #
              source:
                  supplier: null
                  LOT: 
            - name: water
              sumFormula: H2O
              type: solvent
              proportion:
                  value: 50
                  unit: volume percent
              source:
                  refinement: Sartorius Arium 611UV
                  quality: ultrapure water
                  totalOrganicCarbon: 
                      value: 3
                      unit: ppb
            - name: ethanol
              sumFormula: EtOH
              type: solvent
              proportion:
                  value: 50
                  unit: volume percent
            - name: sulfuric acid
              sumFormula: H2SO4
              type: acid
              source:
                  supplier: null
                  LOT: 
              concentration:
                  value: 0.1
                  unit: M #[M, mM, µM, g kg-1, ...] 
        ph: 
            value: 13
            uncertainty: 0.1
        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
            value: 298.15 
            unit: K # [K, C, F]
    electrodes:
        configuration: three # number of electrodes
        referenceElectrode:
            type: RHE # the reference electrode given in the experimental section. Might differ from that in the plot: MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
            source:
                supplier: SI Analytics
        counterElectrode:
            material: Pt
            crystallographicOrientation: poly
            shape: wire
        workingElectrode:
            material: Ni
            crystallographicOrientation: 111 # hkl, 100, 110, poly
            source:
                supplier: null
                LOT: 
            shape: # optional
                type: cylinder # bead/sphere
                height:
                    value: 4
                    unit: mm
                diameter:
                    value: 4
                    unit: mm
            geometricElectrolyteContactArea:  
                value: null
                unit: cm-2
            preparationProcedure: null # Short description of sample preparation. This can be very useful. :)
