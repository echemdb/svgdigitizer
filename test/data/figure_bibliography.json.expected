{
    "resources": [
        {
            "name": "figure_bibliography",
            "type": "table",
            "path": "figure_bibliography.csv",
            "scheme": "file",
            "format": "csv",
            "mediatype": "text/csv",
            "encoding": "utf-8",
            "schema": {
                "fields": [
                    {
                        "name": "t",
                        "type": "number",
                        "unit": "s"
                    },
                    {
                        "name": "T",
                        "type": "number",
                        "unit": "K"
                    },
                    {
                        "name": "j",
                        "type": "number",
                        "unit": "A / m2"
                    }
                ]
            },
            "metadata": {
                "echemdb": {
                    "version": 1,
                    "source": {
                        "doi": "https://doi.org/10.1016/0039-6028(85)90985-9",
                        "citation key": "cv_bibliography",
                        "figure": "2b",
                        "curve": "label",
                        "bibdata": "@article{cv_bibliography,\n    author = \"Engstfeld, Albert K and RÃ¼th, Julian\",\n    title = \"Title\",\n    journal = \"Journal\",\n    volume = \"1\",\n    number = \"1\",\n    pages = \"1--10\",\n    year = \"2022\",\n    abstract = \"Abstract.\"\n}\n"
                    },
                    "curator": {
                        "name": "Albert Engstfeld",
                        "orcid": "https://orcid.org/0000-0002-9686-3948",
                        "digitized": "2021-06-01"
                    },
                    "electrochemical system": {
                        "electrolyte": {
                            "type": "aq",
                            "components": [
                                {
                                    "name": "NaCl",
                                    "sum formula": "NaCl",
                                    "concentration": {
                                        "value": 0.1,
                                        "unit": "M"
                                    },
                                    "type": "salt",
                                    "source": {
                                        "supplier": null,
                                        "LOT": null
                                    }
                                },
                                {
                                    "name": "water",
                                    "sum formula": "H2O",
                                    "type": "solvent",
                                    "proportion": {
                                        "value": 50,
                                        "unit": "volume percent"
                                    },
                                    "source": {
                                        "refinement": "Sartorius Arium 611UV",
                                        "quality": "ultrapure water",
                                        "total organic carbon": {
                                            "value": 3,
                                            "unit": "ppb"
                                        }
                                    }
                                },
                                {
                                    "name": "ethanol",
                                    "sum formula": "EtOH",
                                    "type": "solvent",
                                    "proportion": {
                                        "value": 50,
                                        "unit": "volume percent"
                                    }
                                },
                                {
                                    "name": "sulfuric acid",
                                    "sum formula": "H2SO4",
                                    "type": "acid",
                                    "source": {
                                        "supplier": null,
                                        "LOT": null
                                    },
                                    "concentration": {
                                        "value": 0.1,
                                        "unit": "M"
                                    }
                                }
                            ],
                            "ph": {
                                "value": 13,
                                "uncertainty": 0.1
                            },
                            "temperature": {
                                "value": 298.15,
                                "unit": "K"
                            }
                        },
                        "electrodes": {
                            "configuration": "three",
                            "reference electrode": {
                                "type": "RHE",
                                "source": {
                                    "supplier": "SI Analytics"
                                }
                            },
                            "counter electrode": {
                                "material": "Pt",
                                "crystallographic orientation": "poly",
                                "shape": "wire"
                            },
                            "working electrode": {
                                "material": "Ni",
                                "crystallographic orientation": 111,
                                "source": {
                                    "supplier": null,
                                    "LOT": null
                                },
                                "shape": {
                                    "type": "cylinder",
                                    "height": {
                                        "value": 4,
                                        "unit": "mm"
                                    },
                                    "diameter": {
                                        "value": 4,
                                        "unit": "mm"
                                    }
                                },
                                "geometric electrolyte contact area": {
                                    "value": null,
                                    "unit": "cm-2"
                                },
                                "preparation procedure": null
                            }
                        }
                    },
                    "experimental": {
                        "tags": []
                    },
                    "figure description": {
                        "type": "digitized",
                        "simultaneous measurements": [],
                        "measurement type": "custom",
                        "fields": [
                            {
                                "name": "T",
                                "type": "number",
                                "unit": "K",
                                "orientation": "horizontal"
                            },
                            {
                                "name": "j",
                                "type": "number",
                                "unit": "uA cm-2",
                                "orientation": "vertical"
                            }
                        ],
                        "comment": "",
                        "scan rate": {
                            "value": 50.0,
                            "unit": "K / s"
                        }
                    }
                }
            }
        }
    ]
}
